migration/build-test-runner.md

```diff
@@ -0,0 +1,13 @@
+# Build Test Runner
+
+In Exercism V3, one of the biggest changes is that we'll automatically check if a submitted solution passes all the tests.
+
+We'll check this via a new, track-specific tool: the [_test runner_](https://github.com/exercism/v3-docs/tree/master/anatomy/track-tooling/test-runners). The test runner is track-specific and implemented as a Docker image. Whenever a solution is submitted, a Docker container is created of that image and it runs on the submitted solution. The test runner outputs a JSON file that describes the test results.
+
+The test runner must be able to run the tests suites of both [Concept Exercises](https://github.com/exercism/v3-docs/blob/master/product/concept-exercises.md) and [Practice Exercises](https://github.com/exercism/v3-docs/blob/master/product/practice-exercises.md).
+
+Each track must build a test runner according to [the spec](https://github.com/exercism/v3-docs/blob/master/anatomy/track-tooling/test-runners/interface.md).

```

Comment:
We should ensure that we recommend people use the `generic-test-runner` as a starting point in the docs somewhere.